docs/docs/lib/java.mdx

```diff
@@ -0,0 +1,192 @@
+---
+title: Java SDK
+description: GrowthBook SDK for Java
+sidebar_label: Java
+slug: java
+toc_max_heading_level: 5
+---
+
+import ExternalLink from '@site/src/components/ExternalLink'
+import CodeTabsAddDependencyPartial from '@site/src/partials/java/_code-tabs-add-dependency.mdx'
+import CodeTabsGBContextBuilderPartial from '@site/src/partials/java/_code-tabs-gbcontext-builder.mdx'
+import CodeTabsGBContextConstructorPartial from '@site/src/partials/java/_code-tabs-gbcontext-constructor.mdx'
+import CodeTabsUserAttributesPartial from '@site/src/partials/java/_code-tabs-user-attributes.mdx'
+import CodeTabsUsingFeaturesPartial from '@site/src/partials/java/_code-tabs-using-features.mdx'
+import CodeTabsTrackingCallbackPartial from '@site/src/partials/java/_code-tabs-tracking-callback.mdx'
+import CodeTabsRunExperimentPartial from '@site/src/partials/java/_code-tabs-run-experiment.mdx'
+import CodeTabsExperimentResultCallbackPartial from '@site/src/partials/java/_code-tabs-subscribe-experiment-run.mdx'
+import CodeTabsExperimentResultSubscribeMultipleCallbackPartial from '@site/src/partials/java/_code-tabs-subscribe-multiple.mdx'
+
+This supports Java applications using Java version 1.8 and higher.
+
+:::info Beta
+
+The GrowthBook Java SDK is currently in beta. If you experience any issues, let us know either on [Slack](https://slack.growthbook.io/) or [create an issue](https://github.com/growthbook/growthbook/issues/new/choose).
+
+:::
```

Comment:
I wonder if instead of calling this SDK `Beta`, we should just call it out as being "new".  We can still have this message about reaching out with any issues, but it won't have the negative connotation that "beta" does.

Reply:
Updated to include a green badge that says "new"